app/indexers/hyrax/valkyrie_work_indexer.rb

```diff
@@ -14,6 +14,9 @@ def to_solr # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Met
         solr_doc['generic_type_si'] = 'Work'
         solr_doc['suppressed_bsi'] = suppressed?(resource)
         solr_doc['admin_set_id_ssim'] = [resource.admin_set_id.to_s]
+        admin_set_label = admin_set_label(resource)
+        solr_doc['admin_set_sim']   = admin_set_label
```

Comment:
What is `admin_set_sim` used for?

Reply:
I do not know if `admin_set_sim` is used.  It was [indexed for ActiveFedora works](https://github.com/samvera/hyrax/blob/main/app/indexers/hyrax/work_indexer.rb#L22-L23), so I added it here too.

Reply:
Here is how I believe this is used...
* _tesim - te-tokenized text; s-stored; i-indexed; m-multi-value
* _sim - s-string; i-indexed; m-multi-value

By being indexed, they are both searchable.  By having tokenized and string, they can be searched by token or by the entire string.  Stored is indicating the field is returned as part of the solr document in search results.  It probably doesn't matter which you return, but you only want to return one of them.